src/igz_controls/components/default-dropdown/default-dropdown.component.js

```diff
@@ -182,7 +182,12 @@
 
             // checks if transclude template was passed
             $transclude(function (transclude) {
-                ctrl.isTranscludePassed = transclude.length > 0;
+                ctrl.isTranscludePassed = transclude.length > 0 && !(
+
+                    // a single text node with whitespace only, meaning there is nothing important between the opening
+                    // tag `<default-dropdown-container>` and the closing tag `</default-dropdown-container>`
```

Comment:
there isn't such component as `default-dropdown-container`, should be `igz-default-dropdown`